packages/rulesets/src/native/utilities/arm-helper.ts

```diff
@@ -55,7 +55,9 @@ export class ArmHelper {
 
   private SpecificResourcePathRegEx = new RegExp("/providers/[^/]+(?:/\\w+/default|/\\w+/{[^/]+})+$", "gi")
 
-  private ExtensionResourceReg = new RegExp(".+/providers/.+/providers/.+$", "gi")
+  private ExtensionResourceFullyQualifiedPathReg = new RegExp(".+/providers/.+/providers/.+$", "gi")
+  private ExtensionResourceReg = new RegExp("^/{", "gi")
```

Comment:
Is this enough to ensure the endpoint is an extension resource? Would it be of value to make it something like `^/{\w+}/providers/.+$`?